### ISSUE TITLE
Error starting the app from source on Windows 10 using PowerShell

### ISSUE THREAD

**Issue by bushidocodes** [nsacyber/WALKOFF#206]
I'm testing your documentation is preparation to support the Operation Code hackathon for Windows systems. 

From PowerShell, I did the following:
```
docker run --name walkoff-redis -p 6379:6379 -d redis
choco install python3
python setup_walkoff.py
python walkoff.py
```
This triggered the following output to the PowerShell console:
```
[2018-12-03 10:24:07,642]  root::WARNING - C:\Python36\lib\site-packages\swagger_spec_validator\validator20.py:56: SwaggerValidationWarning:Found "$ref: #/definitions/Uuid" with siblings that will be overwritten. See https://stackoverflow.com/a/48114924 for more information.
[2018-12-03 10:24:07,730]  alembic.runtime.migration::INFO - Context impl SQLiteImpl.
[2018-12-03 10:24:07,730]  alembic.runtime.migration::INFO - Will assume non-transactional DDL.
[2018-12-03 10:24:07,734]  walkoff.cache::INFO - Created redis cache connection
[2018-12-03 10:24:08,739]  walkoff.cache::INFO - Created redis cache connection with options: {}
[2018-12-03 10:24:09,932]  flask.app::INFO - Registering builtin blueprints
[2018-12-03 10:24:09,942]  flask.app::INFO - Registered custom interface blueprint at url prefix /interfaces/HelloWorld
[2018-12-03 10:24:09,947]  flask.app::INFO - Registered custom interface blueprint at url prefix /interfaces/Sample
[2018-12-03 10:24:09,947]  flask.app::INFO - Registered custom interface blueprint at url prefix /interfaces/Sample/<string:action>
[2018-12-03 10:24:09,977]  walkoff::INFO - ***********************************
[2018-12-03 10:24:10,989]  walkoff::INFO - ***** Running WALKOFF v.0.9.2 *****
[2018-12-03 10:24:11,992]  walkoff::INFO - ***********************************
[2018-12-03 10:24:13,157]  root::WARNING - walkoff.py:29: MonkeyPatchWarning:Monkey-patching ssl after ssl has already been imported may lead to errors, including RecursionError on Python 3.6. It may also silently lead to incorrect behaviour on Python 3.7. Please monkey-patch earlier. See https://github.com/gevent/gevent/issues/1016. Modules that had direct imports (NOT patched): ['urllib3.util (C:\\Python36\\lib\\site-packages\\urllib3\\util\\__init__.py)', 'urllib3.contrib.pyopenssl (C:\\Python36\\lib\\site-packages\\urllib3\\contrib\\pyopenssl.py)'].
[2018-12-03 10:24:13,209]  walkoff::WARNING - Cannot find certificates. Using HTTP
Config path .\data\walkoff.config is not a file.
Config path .\data\walkoff.config is not a file.
Config path .\data\walkoff.config is not a file.
[2018-12-03 10:24:14,223]  walkoff::INFO - Listening on host http://127.0.0.1:5000
Config path .\data\walkoff.config is not a file.
```
I navigated to the URL and saw the login screen, but attempting to login with any credentials takes me to to following.

![image](https://user-images.githubusercontent.com/8700322/49382728-2ac07600-f6e5-11e8-86c6-46d27d171bc3.png)

**Comment by adpham95** [comment]
v0.9.3 should fix this issue. When we moved to pre-packaging the front-end components, a couple of libraries were missing that were added in this release.

**However**, we strongly recommend against running WALKOFF directly on Windows, and suggest using a Linux VM or Docker container instead. 

Running WALKOFF directly on Windows has not been strongly supported since v0.9.0 due to changes made in support of a more distributed architecture. It will run on Windows with basic workflow functionality but we do not guarantee stability - the documentation has been updated to more clearly state this.
